accessibility-checker.php

```diff
@@ -27,6 +27,11 @@
 // Include plugin dependency.
 require_once ABSPATH . 'wp-admin/includes/plugin.php';
 
+// Load composer packages.
+if ( file_exists( plugin_dir_path( __FILE__ ) . 'vendor/autoload.php' ) ) {
+	include_once plugin_dir_path( __FILE__ ) . 'vendor/autoload.php';
```

Comment:
Can we limit the loading of this only to the admin? I'm not sure if there is anything that autoloads needed on the frontend.

Reply:
`npm run build` wasn't updating composer. I pushed an update to dev that adds `composer update` and all looks to be working ok now.

Reply:
It's also now only loading vendor/autoload.php if is_admin.